<?xml version="1.0" encoding="UTF-8"?>
<TestRun id="b97a9409-0850-4f45-9d6b-3fa474dcf8f8" name="mshandiliya@TRAINING3 2013-08-03 17:40:02" runUser="PUNE\mshandiliya" xmlns="http://microsoft.com/schemas/VisualStudio/TeamTest/2010">
  <TestSettings name="Local" id="653d5e6b-7427-496f-a05e-e489b8797674">
    <Description>These are default test settings for a local test run.</Description>
    <Deployment enabled="false" runDeploymentRoot="mshandiliya_TRAINING3 2013-08-03 17_40_02" />
    <Execution>
      <TestTypeSpecific />
      <AgentRule name="Execution Agents">
      </AgentRule>
    </Execution>
  </TestSettings>
  <Times creation="2013-08-03T17:40:02.6205758+05:30" queuing="2013-08-03T17:40:05.4338571+05:30" start="2013-08-03T17:40:05.6828820+05:30" finish="2013-08-03T17:42:15.3108435+05:30" />
  <ResultSummary outcome="Error">
    <Counters total="1" executed="1" error="0" failed="0" timeout="0" aborted="1" inconclusive="0" passedButRunAborted="0" notRunnable="0" notExecuted="0" disconnected="0" warning="0" passed="0" completed="0" inProgress="0" pending="0" />
    <RunInfos>
      <RunInfo computerName="TRAINING3" outcome="Error" timestamp="2013-08-03T17:42:15.3088433+05:30">
        <Text>Test host process exited unexpectedly.</Text>
      </RunInfo>
    </RunInfos>
  </ResultSummary>
  <TestDefinitions>
    <UnitTest name="CreateEmployeeSuccess" storage="g:\new folder (3)\employeemanagement.datacontract_assignment\testmodel\bin\debug\testmodel.dll" id="7a467b03-cb85-68f8-8b6c-b475668536cb">
      <Execution id="89455c4a-c2ee-4ffa-9ee3-75cd9bf49ffa" />
      <TestMethod codeBase="G:/New folder (3)/EmployeeManagement.DataContract_Assignment/TestModel/bin/Debug/TestModel.DLL" adapterTypeName="Microsoft.VisualStudio.TestTools.TestTypes.Unit.UnitTestAdapter, Microsoft.VisualStudio.QualityTools.Tips.UnitTest.Adapter, Version=10.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" className="TestModel.ClientTest, TestModel, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null" name="CreateEmployeeSuccess" />
    </UnitTest>
  </TestDefinitions>
  <TestLists>
    <TestList name="Results Not in a List" id="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestList name="All Loaded Results" id="19431567-8539-422a-85d7-44ee4e166bda" />
  </TestLists>
  <TestEntries>
    <TestEntry testId="7a467b03-cb85-68f8-8b6c-b475668536cb" executionId="89455c4a-c2ee-4ffa-9ee3-75cd9bf49ffa" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
  </TestEntries>
  <Results>
    <UnitTestResult executionId="89455c4a-c2ee-4ffa-9ee3-75cd9bf49ffa" testId="7a467b03-cb85-68f8-8b6c-b475668536cb" testName="CreateEmployeeSuccess" computerName="TRAINING3" duration="00:02:09.3989386" startTime="2013-08-03T17:40:05.9119049+05:30" endTime="2013-08-03T17:42:15.3108435+05:30" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Aborted" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="89455c4a-c2ee-4ffa-9ee3-75cd9bf49ffa">
    </UnitTestResult>
  </Results>
</TestRun>